4-star-nosed_mole/api_definitions/rest/ucs.yaml

```
@@ -163,6 +162,15 @@ components:
         - exceptionId
         - data
 
+    PreSignedURL:
+      type: object
+      properties:
+        presigned_url:
+          type: string
+          description: The pre-signed URL for uploading the file part
+      required:
+        - presigned_url
+
```

It would be nice if we could use the same structure as the DRS spec:
```
    AccessURL:
      type: object
      required:
        - url
      properties:
        url:
          type: string
          description: >-
            A fully resolvable URL that can be used to fetch the actual object
            bytes.
        headers:
          type: array
          items:
            type: string
          description: >-
            An optional list of headers to include in the HTTP request to `url`.
            These headers can be used to provide auth tokens required to fetch
            the object bytes.
          example: 'Authorization: Basic Z2E0Z2g6ZHJz'
```
https://editor.swagger.io/?url=https://ga4gh.github.io/data-repository-service-schemas/preview/develop/openapi.yaml

Since the headers are not used in our case, we can simplify it to:
```suggestion
    AccessURL:
      type: object
      required:
        - url
      properties:
        url:
          type: string
          description: >-
            A fully resolvable URL that can be used to fetch the actual object
            bytes.
```